### ISSUE TITLE
fix the metadata updating thing

### ISSUE THREAD

**Issue by mcfiredrill** [datafruits/datafruits#693]
When someone connects lately its always the metadata of the *last* DJ that shows up. 

Probably more of an issue with hotdog_lounge/liquidsoap but wanted to get an issue up.

**Comment by dudethatbe** [comment]
I was looking at the network inspector and was seeing some big requests complete (500kb - 1mb) pretty frequently, I think its the memory leak that we covered before, but I looked at the callstack and thought maybe its related to the front-end component that updates the metadata? Look for "handleUpdatedMediaPlaylist" near the bottom of the screenshot
![Call-stack for media player error](https://user-images.githubusercontent.com/2829501/121971066-dda5cc00-cd45-11eb-9deb-63428e51406f.png)
but idk could be a separate thing altogether

**Comment by mcfiredrill** [comment]
Yeah that's unfortunately video stuff, not related. But thanks for looking though. :tv: @dudethatbe

**Comment by mcfiredrill** [comment]
Seems like the metadata is always correct on the icecast relay, so the issue must be downstream in redis/elixir?
https://streampusher-relay.club/

The current setup is like this:
dj connects to liquidsoap -> liquidsoap hits rails metadata update endpoint that pubs to redis -> hotdog lounge receives metadata through redis and sends to chat

I'll try to make a better ascii diagram later...

**Comment by dudethatbe** [comment]
It seems like this still happening despite the liquidsoap upgrade. And yeah I can also confirm that the icecast relay is always correct (at least when I bother to check it). Maybe we can go over how Redis part of it is inserting and see if we can find any holes @mcfiredrill
